slp2mp4/installer.py

```diff
@@ -32,6 +34,19 @@ def recursive_overwrite(src, dest, ignore=None):
     else:
         shutil.copyfile(src, dest)
 
+
+def patch_dolphin_sys_game_settings():
+    # Remove efb_scale field. This allows selection of resolution options from GFX.ini.
+    gal_ini = os.path.join(FM_WIN_FOLDER, "Sys", "GameSettings", "GAL.ini")
+    gal_ini_parser = configparser.ConfigParser()
+    gal_ini_parser.optionxform = str
+    gal_ini_parser.read(gal_ini)
+    gal_ini_parser.remove_option('Video_Settings', 'EFBScale')
+    gal_ini_fp = open(gal_ini, 'w')
+    gal_ini_parser.write(gal_ini_fp)
+    gal_ini_fp.close()
+
+
```

Comment:
Have to patch the Sys/GameSettings/GAL.ini file here which sets the EFBScale.

Reply:
See comment - this is broken on Linux